hal/src/nRF52840/exflash_hal.c

```diff
@@ -49,6 +50,9 @@ enum qspi_cmds_t {
 
 static const size_t MX25_OTP_SECTOR_SIZE = 4096 / 8; /* 4Kb or 512B */
 
+static bool qspi_initialized = false;;
```

Comment:
Nit: perhaps use a `state` instead of two almost exclusive boolean variables?

Reply:
Fixed